terraform/hcl_mapping.go

```diff
@@ -0,0 +1,41 @@
+package terraform
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcl/v2"
+	gohcl "github.com/hashicorp/hcl/v2/gohcl"
+	hclparse "github.com/hashicorp/hcl/v2/hclparse"
+	structs "github.com/ministryofjustice/cloud-platform-go-library/structs"
+)
+
+func GetTFBody(source string) (hcl.Body, error) {
+	// Parse the HCL file
+	parser := hclparse.NewParser()
+	file, diags := parser.ParseHCLFile(source)
+	if diags.HasErrors() {
+		fmt.Println("error parsing HCL file")
```

Comment:
I don't think we need this log because we are passing out the same message as an error